src/legacy/js/functions/_viewPublishDetails.js

```diff
@@ -7,58 +9,31 @@ function viewPublishDetails(collections) {
         collectionDetails: [],
         pendingDeletes: []
     };
-    var pageDataRequests = []; // list of promises - one for each ajax request to load page data.
-    var onlyOne = 0;
 
-    $.each(collections, function (i, collectionId) {
-        onlyOne += 1;
-        pageDataRequests.push(
-            getCollectionDetails(collectionId,
-                success = function (response) {
-                    if (result.date === manual) {
-                        result.collectionDetails.push({
-                            id: response.id,
-                            name: response.name,
-                            pageDetails: response.reviewed,
-                            datasets: response.datasets,
-                            datasetVersions: response.datasetVersions,
-                            pageType: 'manual',
-                            pendingDeletes: response.pendingDeletes,
-                            interactives: response.interactives
-                        });
-                    } else {
-                        result.collectionDetails.push({
-                            id: response.id,
-                            name: response.name,
-                            pageDetails: response.reviewed,
-                            datasets: response.datasets,
-                            datasetVersions: response.datasetVersions,
-                            interactives: response.interactives
-                        });
-                    }
-                },
-                error = function () {
-                    result.collectionDetails.push({
-                        id: getCollectionIDWithoutUUID(collectionId),
-                        error: true,
-                        pageType: result.date === manual ? "manual" : ""
-                    })
-                }
-            )
-        );
-    });
+    $.each(collections, function(i, collection){
+        if (collection.date === manual) {
+            result.collectionDetails.push({
+                id: collection.id,
+                name: collection.name,
+                pageType: 'manual',
+                showFilesButton: true
+            })
+        } else {
+            result.collectionDetails.push({
+                id: collection.id, 
+                name: collection.name,
+                showFilesButton: true
+            })
+        }
```

Comment:
Since the only difference seems to be the pageType, would something like this be easier to read/maintain?

```suggestion
        result.collectionDetails.push({
            id: collection.id, 
            name: collection.name,
            pageType: collection.date === manual ? "manual" : ""
            showFilesButton: true
       })
```

Reply:
i like it 👍